libs/s25main/mapGenerator/DistanceByProperty.h

```diff
@@ -170,7 +170,7 @@ std::vector<int> DistanceByProperty(const T& args,
         
         queue.pop();
         
-        const auto& neighbors = GridNeighbors(subset[subsetIndex], size);
+        const auto neighbors = GridNeighbors(subset[subsetIndex], size);
         
         for (auto neighbor: neighbors)
```

Comment:
Sometimes it might be worth having the type and not `auto`, e.g. here or for `neighbors`, I guess they are Position or MapPoint? Without an idea its hard to tell, so don't overuse `auto` :)

Reply:
Similar for below `auto subsetIndex = subsetIndexMap.find(neighborIndex);`: The naming isn't great, actually this isn't the index, it is an iterator, so using `itSubsetIndex` as the name would help when reading the code. Of course one could write the iterator type but meh...